### DISCUSSION TITLE
How can i override the default position for custom action from the top right of the page to anywhere else ?!

### DISCUSSION THREAD

**Discussion by KhaldMansour** [filamentphp/filament#6967] [Help]
I have made a custom actions for upload and download in a custom page and they are working fine , but I need to override their positions from the top right corner to be used anywhere else , for example: instead of (add friend , message) buttons which are coming from the blade file of the custom page.
![Screenshot from 2023-07-09 13-28-01](https://github.com/filamentphp/filament/assets/52106756/52df324f-c166-4760-97f4-ca120d3b91b8)
![Screenshot from 2023-07-09 13-27-36](https://github.com/filamentphp/filament/assets/52106756/264e2f2b-2139-48ae-ad24-040cf626e01e)

**Comment by danharrin** [comment]
Not possible in v2, but it has been built into v3
